### PR TITLE
Prepare for geo 0.20.0 release.

### PR DESCRIPTION
**Pull request by frewsxcv**
- [x] I agree to follow the project's [code of conduct](https://github.com/georust/geo/blob/main/CODE_OF_CONDUCT.md).
- [x] I added an entry to `CHANGES.md` if knowledge of this change could be valuable to users.
---

### CODE REVIEW

File: geo/CHANGES.md [hunk 1]
```diff
@@ -55,9 +55,6 @@
   * <https://github.com/georust/geo/pull/629>
 * Implement `Centroid` algorithm on `Geometry` and its remaining variants.
   * <https://github.com/georust/geo/pull/629>
-
```
Comment (michaelkirk) [inline geo/CHANGES.md:58]: 👀

**Review by michaelkirk — COMMENTED** [review]
This LGTM! 

If you want to wait 24hrs, I can commit to having #802 and #721 done, but also it's fine with me if that's in a subsequent release.

File: geo/Cargo.toml [hunk 2]
```diff
@@ -1,8 +1,7 @@
 [package]
 name = "geo"
 description = "Geospatial primitives and algorithms"
-version = "0.19.0"
-authors = ["Corey Farwell <coreyf@rwell.org>"]
```
Comment (michaelkirk) [inline geo/Cargo.toml:5]: 🥲

File: geo/CHANGES.md (hunk 1 above)
Comment (frewsxcv) [reply geo/CHANGES.md:58]: There were two 0.17.1 headings in the changelog. I assume that was an accident so I'm merging them together.

**Comment by frewsxcv** [comment]
> This LGTM!
> 
> If you want to wait 24hrs, I can commit to having #802 and #721 done, but also it's fine with me if that's in a subsequent release.

I will wait!

**Comment by michaelkirk** [comment]
Sorry, I failed 😞 

Comments at https://github.com/georust/geo/issues/721

**Comment by michaelkirk** [comment]
I'd say release whenever you're ready!

**Comment by frewsxcv** [comment]
bors r=michaelkirk

**Pull request merged**